core/src/Revolution/Processors/Security/User/Create.php

```diff
@@ -239,9 +239,16 @@ public function sendNotificationEmail() {
      * @return array|string
      */
     public function cleanup() {
+        $userArray = $this->object->toArray();
+        $profile = $this->object->getOne('Profile');
```

Comment:
As we're only trying to access the `username`, we don't need to load the profile, or even turn it into an array - we can just do `$this->object->get('username')` in the lexicon call a few lines down.